bindata/network/multus-admission-controller/003-webhook.yaml

```diff
@@ -31,6 +31,11 @@ webhooks:
       # On updates, only validate if the Spec changes
       - name: CreateDeleteOrUpdatedSpec
         expression: oldObject == null || object == null || has(object.spec) != has(oldObject.spec) || (has(object.spec) && object.spec != oldObject.spec)
+{{if .OVN_PRE_CONF_UDN_ADDR_ENABLE}}
+      # Ignore default/openshift-ovn-kubernetes NAD to avoid a race between ovn-kubernetes and the multus webhook on install
+      - name: IgnoreDefaultOVNKubernetesNAD
+        expression: object == null || object.metadata.namespace != "openshift-ovn-kubernetes" || object.metadata.name != "default"
```

Comment:
haha in my head i go:

ignore if:
object != null && object.metadata.namespace == "openshift-ovn-kubernetes" && object.metadata.name == "default"

but i see you did exact opp which is the same thing

Reply:
i really hope we can't have a NAD with no namespace and metadata.namespace is never none haha like we hit the hitch with metadata.annotations